src/Units.jl

```diff
@@ -120,4 +120,15 @@ A `Float64` constant equal to 1024`GiB`. Useful for increasing the clarity of sc
 """
 const TiB = 1024GiB
 
+"""
+    struct Time{T}
+        time_in_seconds::T 
+    end
+
+A ``selector'' type to perform indexing in time
+"""
+struct Time{T}
+    time_in_seconds :: T
```

Comment:
```suggestion
    time :: T
```

so that we can generalize to `DateTime` later (where we will need the date not just seconds, in order to interpolate `Vector{DateTime}`)

Reply:
As well as non-dimensional time units